index.html

```diff
@@ -43,6 +43,139 @@ <h3 class="title dtitle">I'm a software developer.</h3>
                 </div> 
             </section>
         </div>
+
+        <section class="recent-works">
+            <div class="recent-works-header">
+                <h2 class="title">My Recent works</h2>
+                <hr class="recent-works-line">
+            </div>
+
+            <div class="cards">
+                <div class="card">
+                    <div class="card-image"></div>
+                    <div class="card-text">
+                        <h3 class="card-title">Multi-Post Stories Gain + Glory</h3>
+                        <ul class="card-tools-list">
+                            <li class="card-tool">Ruby on rails</li>
+                            <li class="card-tool"> CSS</li>
+                            <li class="card-tool">Javascript</li>
+                            <li class="card-tool"> HTML</li>
+                        </ul>
+                        <button type="button" class="see-project btn">See project</button>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-image"></div>
+                    <div class="card-text">
+                        <p class="card-title">Multi-Post Stories Gain + Glory</p>
+                        <ul class="card-tools-list">
+                            <li class="card-tool">Ruby on rails</li>
+                            <li class="card-tool"> CSS</li>
+                            <li class="card-tool">Javascript</li>
+                            <li class="card-tool"> HTML</li>
+                        </ul>
+                        <button type="button" class="see-project btn">See project</button>
```

Comment:
- [ ] Great job adding the button effects as requested by the previous reviewer! However, I noticed that the `color` you used for the effects is different from the green color used in the template. Please remember to use the correct green color for the button effects to ensure consistency with the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-468&t=bze6xl2nLShZtj6i-0).

### Colors:
![image](https://user-images.githubusercontent.com/107952724/235332166-4243272d-b597-4461-a2b8-e369b4ecf520.png)